README.md

```diff
@@ -1,8 +1,362 @@
 # CUDA C++ Core Libraries (CCCL)
 
-The future home of the CUDA C++ Core Libraries (Thrust, CUB, libcu++).
+Welcome to the CUDA C++ Core Libraries (CCCL) where our mission is to make CUDA C++ more delightful.
+
+This repository unifies three essential CUDA C++ libraries into a single, convenient repository: 
+
+1. [Thrust](https://github.com/nvidia/thrust)
```

Comment:
It might not be helpful to link to the _previous_ homes of these libraries. It makes it look like those repos will still be maintained. I like Mark's suggestion of using a bulleted list, and remove the links.

Reply:
I have a different opinion. I like a bullet list too, but keep the links because users who are familiar with the old repos would wonder where the new changes reside now. (When archiving the old repos I am sure we'd post the announcement in the READMEs there pointing to CCCL, but it's complementary to the changes in discussion here.)

Reply:
I like re-pointing them to the subdirectories, or to the user guide for each. I don't like pointing them at deprecated repositories, that is confusing.